app/views/items/show.html.erb

```diff
@@ -1,17 +1,27 @@
 <% content_for :sidebar do %>
-  <%= sidebar_button_link 'Return to Listing', items_path %>
   <%= edit_item_button %>
   <%= add_repair_button %>
   <%= add_inspection_button %>
 <% end %>
 
 <h3>
+  <%= link_to @item.item_category,
+                item_category_path(@item.item_category) %> >
+  <%= link_to @item.item_type, @item.item_type %> >
```

Comment:
this just looks like a typo.  It would look less like a typo if it was

    link_to @item.item_type.name, @item.item_type

and even better if it was 

    link_to @item_type.name, item_type_path(@item.item_type)

This would also be more consistent with how you're making links elsewhere.